contexts/arbo-context/arbo-filter-update-steps.tsx

```diff
@@ -0,0 +1,29 @@
+import { MapRef } from "react-map-gl"
+import { ArboAction, ArboStateType } from "./arbo-context"
+import { pipe } from "fp-ts/lib/function.js";
+import { adjustMapPositionStep } from "./filter-update-steps/adjust-map-position-step";
+import { updatePediatricAgeGroupFilter } from "./filter-update-steps/update-pediatric-age-group-filter";
+import { addActionToSelectedFiltersStep } from "./filter-update-steps/add-action-to-selected-filters-step";
+import { applyNewSelectedFiltersStep } from "./filter-update-steps/apply-new-selected-filters-step";
+
+export interface HandleArboFilterUpdateInput {
+  state: ArboStateType,
+  action: ArboAction,
+  map: MapRef | undefined
+}
+
+export interface HandleArboFilterUpdateOutput {
+  state: ArboStateType,
+  action: ArboAction,
+  map: MapRef | undefined
+}
+
+export const handleArboFilterUpdate = (input: HandleArboFilterUpdateInput): HandleArboFilterUpdateOutput => {
+  return pipe(
```

Comment:
https://gcanti.github.io/fp-ts/modules/function.ts.html#pipe
Equivalent to this:
```
applyNewSelectedFiltersStep(
  updatePediatricAgeGroupFilter(
    adjustMapPositionStep(
      addActionToSelectedFiltersStep(
        input
      )
    )
  )
)
```
Just reads better if you have a lot of functions and lists the functions in the order they are called not reverse the order they are called which I'm a big fan of.

This is what the ETL uses too and I really love this function.

Reply:
+1 Yeah this is super cool! I did not know this existed